Fix: Show the right type inside the no history message [ED-12342]

pkniazevych opened the pull request:
Replaced all `promptType` variable names with `historyType` as it makes more sense and saves consistency between history type constants and variable names.

pkniazevych committed:
Fix: Show the right type inside the "no history" message [ED-12342]

aviranLevi1 commented on modules/ai/assets/js/editor/components/prompt-history/parts/modal-upgrade.js:
```diff
@@ -1,24 +1,34 @@
+	// Translators: %s: History type.
+	[ VARIANT_FULL ]: __( 'Want to see your %s generation history for as far as the past 90 days?', 'elementor' ),
+	// Translators: %s: History type.
+	[ VARIANT_SMALL ]: __( 'Want to see your %s generation history for the past 90 days?', 'elementor' ),
```
I suggest you to move the messages to consts/enum file

pkniazevych replied:
They're actually needed only here, so I think keeping them as constants in the same file should work for us 🙂

Nevoss committed:
Fix: Widget panel is empty for Editor role with 'Access to edit content only' (#23939)

TzviRabinovitch committed:
Internal: Change Workflow Permissions [ED-12345] (#23942)

rotemee committed:
Internal: Adjust functionality of upload SVG control [ED-12192] (#23937) (#23945)

rotemee committed:
Internal: Adjust functionality of upload SVG control [ED-12192-2] (#23951)

davseve committed:
Internal: Change-log for 3.16.5 [ED-12411] (#23987)

arielk committed:
Tweak: Changed Gutenberg asset loading experiment to Optimized Gutenberg Loading (#23994)

for better explanation  CP [ED-12391]

[ED-12391]:
https://elementor.atlassian.net/browse/ED-12391?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

KingYes committed:
Internal: Fix typo on description [ED-12391] (#23997)

hein-obox committed:
Fix: Rating icons are not breaking into lines - beta [ED-12386] (#24000)

arielk committed:
Merge branch 'main' into fix/ED-12342

arielk committed:
Merge branch '3.17' into fix/ED-12342

pkniazevych committed:
Fix: Decrease prompt history button's margin [ED-12343] (#23965)

Co-authored-by: TzviRabinovitch <99259629+TzviRabinovitch@users.noreply.github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>
Co-authored-by: Nevo Golan <20494189+Nevoss@users.noreply.github.com>
Co-authored-by: rotemee <61740845+rotemee@users.noreply.github.com>
Co-authored-by: Louis Wolmarans <louiswol94@gmail.com>
Co-authored-by: Davids Seveloff <94905340+davseve@users.noreply.github.com>
Co-authored-by: Ariel Klikstein <ariel@elementor.com>
Co-authored-by: Yakir Sitbon <kingyes1@gmail.com>
Co-authored-by: Hein van Vlastuin <94352322+hein-obox@users.noreply.github.com>

arielk committed:
Merge branch 'main' into fix/ED-12342

arielk committed:
Merge branch '3.17' into fix/ED-12342
